### PR TITLE
Enable viewing saved shows in global saves/follows view

### PR DESCRIPTION
**Pull request by ashleyjelks**
This PR exposes the Shows tab in the Favorites screen and enables users to view all their saved shows there. 

[Links to LD-194](https://artsyproduct.atlassian.net/browse/LD-194)

kapture 2019-02-22 at 10 40 31

#skip_new_tests

### CODE REVIEW

File: src/lib/Components/Lists/SavedShowItemRow.tsx [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+  status: string
+  images: {
+    url: string
+  }
+  partner: {
+    name: string
+  }
+  is_followed: boolean
+  relayEnvironment: RelayProp
+}
```
Comment (orta) [inline src/lib/Components/Lists/SavedShowItemRow.tsx]: I feel like this should be generated for you somewhere?

Comment (ashfurrow) [reply src/lib/Components/Lists/SavedShowItemRow.tsx]: I think it's a nested part of the type generated by a fragment, so I don't think it's accessible from a "this object is this shape" perspective. 

A better approach could be to define the fragment in `SavedFairItemRow` and then spread across that fragment in the `SavedFairs` component instead. I'm working in this code for analytics, so I can take on that refactor after this PR is merged.

Comment (l2succes) [reply src/lib/Components/Lists/SavedShowItemRow.tsx]: Yup that'd be preferable indeed. sounds good @ashfurrow

**Comment by l2succes** [comment]
I agree we should have a relay container for `SavedShowItemRow` but we can merge as is for now and tackle during analytics work

**Pull request merged**